### ISSUE TITLE
Project Overview

### ISSUE THREAD

**Issue by hmontero1205** [bbqmatt/capstone-expresso#1]
Overall, your work meets the project specifications -- awesome work!!

I actually just don't have any issues with your code. Spot on!
My only suggestion for you is to comment your code. You need to be able to effectively document your work so that reviewers can easily follow along and understand what you're doing.

Keep it up :)

**Comment by bbqmatt** [comment]
Yeah - I just got lazy towards the end. I did have enormous amounts of problems working out parameter naming. But I sorted it out by running the chrome inspect debugger. thanks for the feedback.
